spec/acceptance/fail2ban_spec.rb

```diff
@@ -0,0 +1,237 @@
+# frozen_string_literal: true
+
+require 'spec_helper_acceptance'
+
+case fact('os.family')
+when 'Debian'
+  package_name          = 'fail2ban'
+  config_file_path      = '/etc/fail2ban/jail.conf'
+  service_name          = 'fail2ban'
+  ssh_log_file          = '/var/log/auth.log'
+  ssh_jail              = 'sshd'
+when 'RedHat'
+  package_name          = 'fail2ban'
+  config_file_path      = '/etc/fail2ban/jail.conf'
+  service_name          = 'fail2ban'
+  ssh_log_file          = '/var/log/secure'
+  ssh_jail              = 'ssh'
+end
+
+# Ensure the ssh log file is created, otherwise the service doesn't start completely
+shell("touch #{ssh_log_file}")
+
+describe 'fail2ban' do
+  context 'with defaults' do
+    it_behaves_like 'an idempotent resource' do
+      let(:manifest) do
+        <<-EOS
+      class { 'fail2ban': }
```

Comment:
This should be indented instead of outdented.  I'm surprised that rubocop didn't warn on this...